libs/core-components/src/lib/Form/FileInputDropZone.jsx

```diff
@@ -0,0 +1,121 @@
+/* FP-993: Allow use by DataFilesUploadModal */
+import React, { useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import Button from '../Button';
+import InlineMessage from '../Message';
+import PropTypes from 'prop-types';
+import './FileInputDropZone.scss';
```

Comment:
Requests:

1. Rename to `FileInputDropZone.global.css`. [Reference.](https://github.com/TACC/tup-ui/blob/task/tup-ui-styles/apps/tup-ui/src/styles/README.md)
2. Convert SCSS syntax—if any—to native CSS or  [PostCSS syntax](https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#glossary).